### PR TITLE
Deploy to DigitalOcean 

### PR DESCRIPTION
**Pull request by coinbookbrasil2021**
- Folder .do

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 ![docker](https://img.shields.io/docker/pulls/edeng23/binance-trade-bot)
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/edeng23/binance-trade-bot)
 
+[![Deploy to DO](https://mp-assets1.sfo2.digitaloceanspaces.com/deploy-to-do/do-btn-blue.svg)](https://cloud.digitalocean.com/apps/new?repo=https://github.com/coinbookbrasil/binance-trade-bot/tree/master&refcode=a076ff7a9a6a)
```
Comment (DmytroLitvinov) [inline README.md:7]: Hi @edeng23 ,
First of all there is a `refcode` - I don't mind it but better to say about it somewhere.
And also repo URL after merge should go to your repo, not @coinbookbrasil.

File: .do/deploy.template.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+spec:
+  name: binance-trade-bot
+  workers:
+  - environment_slug: python
+    git:
+      branch: master
+      repo_clone_url: https://github.com/coinbookbrasil/binance-trade-bot.git
```
Comment (DmytroLitvinov) [inline .do/deploy.template.yaml:7]: @edeng23 , same. URL probably should be `https://github.com/edeng23/binance-trade-bot.git`

Comment (coinbookbrasil2021) [reply .do/deploy.template.yaml:7]: Sorry. forgot to change

Comment (coinbookbrasil2021) [reply .do/deploy.template.yaml:7]: I updated the pull. Thanks!

**Pull request merged**